protocols/relay/src/behaviour/handler.rs

```diff
@@ -59,12 +62,12 @@ pub enum In {
     },
     DenyReservationReq {
         inbound_reservation_req: inbound_hop::ReservationReq,
-        status: proto::Status,
+        status: Status,
```

Comment:
Please try to avoid diffs like these :)

We extensively use the pattern of referencing symbols with 1-2 module names for disambiguation.